scripts/plot_network_catalogue.py

```diff
@@ -170,7 +170,7 @@ def prettify_network_name(name: str) -> str:
         # Create the map
         zoom_start = 12 if network_name.startswith("MV") else 16
         title = prettify_network_name(name=network_name)
-        m = RoseauMap(location=list(reversed(buses_gdf.unary_union.centroid.coords[0])), zoom_start=zoom_start)
+        m = RoseauMap(location=list(reversed(buses_gdf.union_all().centroid.coords[0])), zoom_start=zoom_start)
```

Comment:
@benoit9126 `GeoSeries.unary_union` started emitting a deprecation warning. You may want to change it all over our code base when you see it used.